### PR TITLE
Draw aura textures underneath tokens

### PR DESCRIPTION
**Pull request by stwlam**
[swimmers](https://github.com/foundryvtt/pf2e/assets/106829671/146a0f6c-e390-485e-a656-84a164588041)

### CODE REVIEW

File: src/module/canvas/token/aura/renderer.ts [hunk 1]
```diff
@@ -97,29 +106,40 @@ class AuraRenderer extends PIXI.Graphics implements TokenAuraData {
...
-            const video = game.video.getVideoSource(this.texture) ?? globalVideo;
-            video.playbackRate = data.playbackRate;
-            const offset = data.loop ? Math.random() * video.duration : 0;
-            game.video.play(video, { volume: 0, offset, loop: data.loop });
-        }
+        const texture = await (async (): Promise<PIXI.Texture | null> => {
+            const maybeTexture = await loadTexture(data.src, { fallback: "icons/svg/hazard.svg" });
+            if (!(maybeTexture instanceof PIXI.Texture)) return null;
+
+            const globalVideo = isVideoFilePath(data.src) ? game.video.getVideoSource(maybeTexture) : null;
```
Comment (CarlosFdez) [inline src/module/canvas/token/aura/renderer.ts:117]: given maybeTexture is getting destroyed anyways, would it not be better to make the entire image texture load part of what happens when there is no video texture?

Comment (stwlam) [reply src/module/canvas/token/aura/renderer.ts:117]: This didn't change in the PR and was pulled from core. For some reason it's necessary?

Comment (CarlosFdez) [reply src/module/canvas/token/aura/renderer.ts:117]: I misread the code, getVideoSource is pulled from maybeTexture already.

File: src/module/canvas/token/aura/renderer.ts [hunk 2]
```diff
@@ -97,29 +106,40 @@ class AuraRenderer extends PIXI.Graphics implements TokenAuraData {
...
         const diameter = radius * 2;
-        const scale = { x: diameter / this.texture.width, y: diameter / this.texture.height };
+        const scale = { x: diameter / texture.width, y: diameter / texture.height };
         const matrix = new PIXI.Matrix(scale.x, undefined, undefined, scale.y, radius, radius);
-        this.beginTextureFill({ texture: this.texture, alpha: data.alpha, matrix }).drawCircle(0, 0, radius).endFill();
+        this.textureContainer = new PIXI.Graphics()
+            .beginTextureFill({ texture, alpha: data.alpha, matrix })
+            .drawCircle(0, 0, radius)
+            .endFill();
+        this.textureContainer.position.set(this.token.center.x, this.token.center.y);
```
Comment (CarlosFdez) [inline src/module/canvas/token/aura/renderer.ts:140]: Still looks good with tiny tokens?

**Pull request merged**